src/templates/implementer-incomplete.js

```
@@ -0,0 +1,70 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+
+import Layout from '../components/layout'
+import SEO from '../components/seo'
+import Note from '../components/note'
+import ListOfImplementations from '../components/list-of-implementations'
+import { filterByConsistency } from './implementer'
+
+import './implementer-incomplete.scss'
+
+const ImplementerIncomplete = ({ location, data }) => {
+	const { title, implementerData } = data.sitePage.context
+	const implementerReport = JSON.parse(implementerData)
+
+	const areAllMappingsIncomplete = implementerReport.actMapping.every(({ complete }) => complete === false)
+	const incompleteMaps = filterByConsistency(implementerReport.actMapping, ['inconsistent'])
+
+	if (areAllMappingsIncomplete) {
+		return (
+			<Layout location={location}>
+				<SEO title={title} />
+				<section className="page-implementer-incomplete">
+					<h1>{title}</h1>
+					<Note
+						cls={`invalid`}
+						title={`Incomplete Implementation`}
+						body={`All implementations provided are incomplete. Kindly submit an amended implementation report.`}
+					/>
+				</section>
+			</Layout>
+		)
+	}
```

I'm not fully sure about how all of this work, but this seems odd in the "incomplete mapping" template.

Given that we are here trying to display the incomplete mappings, we should just use the default case, not have a special message to hide them.